### DISCUSSION TITLE
Setting Pan to left click

### DISCUSSION THREAD

**Discussion by phobiusmobius** [sumatrapdfreader/sumatrapdf#3605] [Q&A] [answered]
I'd like to have Panning default to the left mouse button (without selecting when hovering over text). Has anyone done that before?

**Comment by GitHubRulesOK** [comment]
Old Answer was reverse the mouse buttons so left handed user gets pan on left and click on right !
If you wanted that for only one application then I recommended programming per application which if you use a mouse without that ability then use x-mouse driver https://github.com/sumatrapdfreader/sumatrapdf/issues/809#issuecomment-430397570, the alternative was AutoHotKey.
 
HOWEVER with latest pre release unsure if SumatraPDF can re-order the buttons or not, I cant find any such related functions

**Reply by phobiusmobius** [reply, marked as answer]
I figured out a pretty good and easy solution : 
1 - Download Key Manager. 
2 - Swap mouse keys ONLY in Fullscreen mode (so you can select and click normally in the rest of the app).

![Screenshot (11)](https://github.com/sumatrapdfreader/sumatrapdf/assets/141643364/3de7b338-9c37-4d15-a31f-3c4bf97310dd)

Bonus Remaps: 
A - Map Scroll up and down in fullscreen mode to zoom in and out respectively.
B - Map Mouse-1,2, or any programmable button, and srcoll up and down to Page up and Page down.
C - Set bookmarks from F12 to F1 for easier reach : )
